src/stream.rs

```diff
@@ -465,6 +465,15 @@ impl CompressParams {
         self.lgblock = lgblock;
         self
     }
+
+    /// Get the current block size
+    pub fn get_lgblock(&self) -> usize {
+        1usize << self.lgblock
+    }
+    /// Get the current window size
+    pub fn get_lgwin(&self) -> usize {
```

Comment:
Perhaps this could return the `u32` that's set above? It's a little odd that you set a different value than you'd get?